plugins/modules/keycloak_realm_info.py

```diff
@@ -26,7 +26,12 @@
       be returned that way by this module. You may pass single values for attributes when calling the module,
       and this will be translated into a list suitable for the API.
 
+attributes:
+    action_group:
+        version_added: 11.0.0
+
 extends_documentation_fragment:
+    - community.general.keycloak.actiongroup_keycloak
     - community.general.attributes
     - community.general.attributes.info_module
```

Comment:
Hmm, this module is exceptional in that it does not use the docs fragment and the argument spec from the module utils. Basically it only has the `auth_keycloak_url` and `validate_certs` options that are shared by the other modules. For that reason it should probably be excluded from the action group, since otherwise you cannot use the action group when using this module (except for these two options, but not for all other shared ones).

It's probably best to skip it here, and once someone makes the module use the module utils' argument spec and docs fragment, to then also add it to the action group.

Reply:
good catch, excluded this one now